### PR TITLE
#867 add AR notes to volumes_daily

### PR DESCRIPTION
**Pull request by gabrielwol**
## What this pull request accomplishes:
- Adds context from `anomalous_range` table to filtered `volumes_daily` (see sample in the comment below)
- new columns `anomalous_range_caveats` and `anomalous_range_uid` are arrays to support multiple issues per row (last row below)

## Issue(s) this solves:
- #867 

## What, in particular, needs to reviewed:
- What do you think of the formatting of the anomalous_ranges and the information contained in the extra columns? We can also consider using `string_agg` instead of `array_agg`. 
- You can see this in action at `gwolofs.volumes_daily`. Already confirmed that:
```sql
SELECT (SELECT COUNT(*) FROM gwolofs.volumes_daily) = (SELECT COUNT(*) FROM miovision_api.volumes_daily)
```

## What needs to be done by a sysadmin after this PR is merged
- DROP existing view and replace with new definition.
- After changes are approved I will update the readme.

### CODE REVIEW

**Comment by gabrielwol** [comment]
Preview formatted better here: 

| "dt"         | "intersection_uid" | "classification_uid" | "daily_volume" | "isodow" | "holiday" | "datetime_bins_missing" | "unacceptable_gap_minutes" | "avg_historical_gap_vol" | "anomalous_range_caveats"                                                                                                                                                                                                                                                             | "anomalous_range_uid" |
|--------------|--------------------|----------------------|----------------|----------|-----------|-------------------------|----------------------------|--------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------|
| "2022-01-27" | 65                 | 1                    | 5658           | 4        | false     | 342                     | 0                          |                          | "{NULL}"                                                                                                                                                                                                                                                                              | "{NULL}"              |
| "2022-01-27" | 65                 | 2                    | 4              | 4        | false     | 1436                    | 0                          |                          | "{NULL}"                                                                                                                                                                                                                                                                              | "{NULL}"              |
| "2022-01-27" | 65                 | 3                    | 18             | 4        | false     | 1422                    | 0                          |                          | "{""From this point forward, this category includes streetcars, where before it did not.""}"                                                                                                                                                                                          | "{122}"               |
| "2022-01-27" | 65                 | 4                    | 110            | 4        | false     | 1343                    | 0                          |                          | "{NULL}"                                                                                                                                                                                                                                                                              | "{NULL}"              |
| "2022-01-27" | 65                 | 9                    | 2              | 4        | false     | 1438                    | 0                          |                          | "{""From this point forward, this classification shoul no longer include streetcars where before it did""}"                                                                                                                                                                           | "{123}"               |
| "2022-01-27" | 65                 | 10                   | 9              | 4        | false     | 1431                    | 0                          |                          | "{""Volumes might be low because of an algorithm issue during this range. Volumes for bikes in crosswalks (classification_uid 7) were present in the API but excluded from the database. "",""Mioivision incorrectly idenitified bikes as pedestrians, those records are removed.""}" | "{893,897}"           |

File: volumes/miovision/sql/views/create-view-volumes_daily.sql [hunk 1]
```diff
@@ -8,10 +8,16 @@ CREATE VIEW miovision_api.volumes_daily AS (
         v.holiday,
         v.datetime_bins_missing,
         v.unacceptable_gap_minutes,
-        v.avg_historical_gap_vol
+        v.avg_historical_gap_vol,
+        array_agg(ar.notes ORDER BY ar.range_start, ar.uid)
```
Comment (chmnata) [inline volumes/miovision/sql/views/create-view-volumes_daily.sql:12]: seems like if there are no notes the value would look like `{NULL}`, should we make it actually null with 
`array_agg(ar.notes ORDER BY ar.range_start, ar.uid) filter(where notes is not null) ` instead?

Comment (gabrielwol) [reply volumes/miovision/sql/views/create-view-volumes_daily.sql]: Good suggestion! Also added column comments for those columns.

**Review by chmnata — APPROVED** [review]

**Comment by chmnata** [comment]
Merging! @gabrielwol you can make the changes in db :meow_ez:

**Comment by gabrielwol** [comment]
✅ with help from Nate who had to drop some dependencies.

**Pull request merged**